frontend/package.json

```diff
@@ -27,60 +27,60 @@
   },
   "dependencies": {
     "@bcgov/bc-sans": "^2.1.0",
-    "@fortawesome/fontawesome-svg-core": "^6.6.0",
-    "@fortawesome/free-solid-svg-icons": "^6.6.0",
+    "@fortawesome/fontawesome-svg-core": "^6.7.1",
+    "@fortawesome/free-solid-svg-icons": "^6.7.1",
     "@fortawesome/vue-fontawesome": "^3.0.8",
     "@geoman-io/leaflet-geoman-free": "^2.17.0",
     "@types/geojson": "^7946.0.14",
-    "@types/leaflet": "^1.9.12",
+    "@types/leaflet": "^1.9.14",
     "@types/proj4": "^2.5.5",
-    "axios": "^1.7.2",
-    "date-fns": "^3.6.0",
-    "filesize": "^10.1.4",
+    "axios": "^1.7.8",
+    "date-fns": "^4.1.0",
+    "filesize": "^10.1.6",
     "geojson": "^0.5.0",
     "leaflet": "^1.9.4",
-    "oidc-client-ts": "^3.0.1",
-    "pinia": "^2.2.2",
-    "pinia-plugin-persistedstate": "^3.2.1",
+    "oidc-client-ts": "^3.1.0",
+    "pinia": "^2.2.7",
+    "pinia-plugin-persistedstate": "^4.1.3",
     "primeflex": "^3.3.1",
-    "primeicons": "^6.0.1",
+    "primeicons": "^7.0.0",
     "primevue": "3.53.0",
-    "proj4": "^2.11.0",
-    "qrcode.vue": "^3.4.1",
+    "proj4": "^2.15.0",
+    "qrcode.vue": "^3.6.0",
     "quill": "^2.0.2",
-    "uuid": "^10.0.0",
-    "vee-validate": "^4.13.2",
-    "vue": "^3.5.4",
+    "uuid": "^11.0.3",
+    "vee-validate": "^4.14.7",
+    "vue": "^3.5.13",
     "vue-i18n": "^10.0.4",
-    "vue-router": "^4.4.4",
+    "vue-router": "^4.5.0",
     "yup": "^1.4.0"
   },
   "devDependencies": {
-    "@pinia/testing": "^0.1.3",
+    "@pinia/testing": "^0.1.7",
     "@testing-library/vue": "^8.1.0",
-    "@tsconfig/node18": "^18.2.4",
-    "@types/node": "^20.14.11",
+    "@tsconfig/node22": "^22.0.0",
+    "@types/node": "^22.10.0",
     "@types/uuid": "^10.0.0",
-    "@vitejs/plugin-vue": "^5.1.3",
-    "@vitejs/plugin-vue-jsx": "^4.0.0",
-    "@vitest/coverage-istanbul": "^2.0.3",
+    "@vitejs/plugin-vue": "^5.2.1",
+    "@vitejs/plugin-vue-jsx": "^4.1.1",
+    "@vitest/coverage-istanbul": "^2.1.6",
     "@vue/eslint-config-typescript": "^13.0.0",
     "@vue/test-utils": "^2.4.6",
-    "@vue/tsconfig": "^0.5.1",
+    "@vue/tsconfig": "^0.6.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-vitest-globals": "^1.5.0",
-    "eslint-plugin-vue": "^9.27.0",
-    "happy-dom": "^14.12.3",
-    "prettier": "^3.3.3",
+    "eslint-plugin-vue": "^9.31.0",
+    "happy-dom": "^15.11.7",
+    "prettier": "^3.4.1",
     "rimraf": "^6.0.1",
-    "sass": "^1.77.8",
+    "sass": "^1.81.0",
     "ts-node": "^10.9.2",
-    "typescript": "^5.5.3",
-    "vite": "^5.4.4",
-    "vite-plugin-vue-devtools": "^7.4.5",
-    "vitest": "^2.0.3",
+    "typescript": "~5.6.2",
+    "vite": "^6.0.1",
+    "vite-plugin-vue-devtools": "7.6.5",
+    "vitest": "^2.1.6",
     "volar-service-eslint": "^0.0.17",
-    "vue-tsc": "^2.1.6"
+    "vue-tsc": "~2.0.29"
```

Comment:
looks like version has gone down, is this intentional? am I reading it wrong?

Reply:
There is a temporary incompatibility between typescript and vue-tsc. There is a patch for vue-tsc that will resolve the issue next release. See this issue for details: https://github.com/vuejs/language-tools/issues/5018